Karma/Areas/Identity/Pages/Account/Manage/Index.cshtml

```diff
@@ -1,7 +1,6 @@
 ﻿@page
 @model IndexModel
 @{
-    ViewData["Title"] = "Profile";
```

Comment:
Is page' title set somewhere else? If not, then why are we removing the title?

Reply:
Yes Profile page Title is set in Views/Profile/Index.cshtml